db-checkbook/build_scripts/04_summarization.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+import pandas as pd
+
+_curr_file_path = Path(__file__).resolve()
+LIB_DIR = _curr_file_path.parent.parent / '.library'
+final_categorization_file = pd.read_csv(_curr_file_path.parent.parent / 'output' / 'temp_historical_spend.csv')
```

Comment:
loading relatively large data should probably be in a function rather than at the top of the script. maybe at the start of `categorization_run_summary_statistics`?